docs/content/preview/api/ysql/syntax_resources/ysql_grammar.ebnf

```diff
@@ -211,6 +211,7 @@ alter_table_action =
   ( 'ADD' [ 'COLUMN' ] column_name data_type [ alter_column_constraint { alter_column_constraint } ]
     | 'RENAME' 'TO' table_name
     | 'DROP' [ 'COLUMN' ] column_name [ 'RESTRICT' | 'CASCADE' ]
+    | alter_column_type
```

Comment:
subrule is unconventional here, so should avoid.  Also, "alter_column_type" is unrelated to "alter_column_constraint" even though they have same prefix, so it is confusing.